### PR TITLE
Update

### PR DESCRIPTION
**Pull request by JedMeister**
sry about all the noise... This should be good to go... 

double check you are happy with my edited changelog; just tried to make it consistent with other apps

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+- Laravel 
+  
+   - Installed from upstream source code to /var/www/laravel.
+   - Bower to manage dependencies of your front-end package (jQuery, Bootstrap, AngularJS, ...)
+   - Composer globaly installed for all your projects
+   - NPM to allow to easily install Yeoman, Grunt, ...
+
 - SSL support out of the box.
-- `PHPMyAdmin`_ administration frontend for MySQL (listening on port 12322 - uses SSL).
+- `PHAdmin`_ administration frontend for MySQL (listening on port 12322 - uses SSL).
```
Comment (qrntz) [inline README.rst]: PHAdmin?

**Comment by qrntz** [comment]
@JedMeister, thanks!

Actually, I'm afraid the README will have to be updated some more (it mentions e. g. Bower and NPM which are no longer needed nor included)… I sort of missed that and will work on it.

Do you mind if I do a bit of Git thaumaturgy to consolidate your changes instead of merging as-is?

**Comment by JedMeister** [comment]
@qrntz I would LOVE it if you did some "Git thaumaturgy" (had to look that one up) to tidy it up! 

I was having a bad day yesterday... Not enough sleep and way too much coffee. So instead of just being a bit vague and scattered; I was hyperspeed vague and scattered...! Feeling better today! :smile:

**Comment by qrntz** [comment]
@JedMeister, no worries. I've applied your changes and fixed the loose ends. Also, as far as I understand, we'll include my copy of the repo for release (with due credit to @kefniark) so makes sense to merge update into master (which I just did).

**Comment by JedMeister** [comment]
Great work mate! :smile:

**Pull request closed without merging**